src/App/Fossa/VSI/DynLinked/Util.hs

```diff
@@ -31,6 +37,10 @@ hasSetUID file = do
   stat <- fatalOnSomeException "get file status" . sendIO . getFileStatus $ P.toFilePath file
   pure $ fileMode stat .&. setUserIDMode /= 0
 
+-- | The root of the primary file system.
+fsRoot :: Path Abs Dir
+fsRoot = $(mkAbsDir "/")
+
 #endif
 
 runningLinux :: Bool
```

Comment:
I'm probably going to extract `fsRoot` and a set of `runningLinux`-like helpers into a higher-level package in a future PR. They seem really helpful for cross-platform work.